### PR TITLE
chore: ensure tls required on s3 buckets

### PR DESCRIPTION
**Pull request by TimJones**
Ensure that non-TLS connections are denied to S3 buckets via policy. Required for compliance.

### CODE REVIEW

File: hack/cloud-image-uploader/aws.go [hunk 1]
```diff
@@ -139,6 +140,42 @@ func (au *AWSUploader) registerAMI(ctx context.Context, region string, svc *ec2.
 		}
 	}()
 
+	denyInsecurePolicy := map[string]interface{}{
```
Comment (frezbo) [inline hack/cloud-image-uploader/aws.go:143]: let's just define this as a var and just do `fmt.Sprintf` and use that as string

Comment (TimJones) [reply hack/cloud-image-uploader/aws.go]: Done, but imo that looks way worse :sweat_smile:

Comment (frezbo) [reply hack/cloud-image-uploader/aws.go]: oh  I meant define as a `var ()` block at the top, but nothing critical here

**Review by frezbo — APPROVED** [review]

File: hack/cloud-image-uploader/aws.go (hunk 1 above)
Comment (TimJones) [reply hack/cloud-image-uploader/aws.go]: OK, I redid that then :)

**Review by frezbo — APPROVED** [review]

**Pull request merged**